std-bits/base/src/main/java/org/enso/base/Regex_Utils.java

```diff
@@ -70,4 +70,42 @@ public static String[] find_all_matches(String regex, String text) {
     }
     return allMatches.toArray(new String[0]);
   }
+
+  /**
+   * Converts a SQL-like pattern into a Regex with the same semantics.
+   *
+   * <p>Special regex characters present in the input pattern are quoted to match them literally
+   * according to the SQL-like format.
+   */
+  public static String sql_like_pattern_to_regex(String sql_pattern) {
```

Comment:
think the pattern needs to include `^` and `$` to indicate the start and end of the value.

`hello_` ==> `^hello_$` as SQL like patterns mean that.

Reply:
As explained below, I already test that the LIKE matches only the whole string.

I prefer to use the specific 'match whole string' regex function and keep the expression itself free of ^$ because they can have different meaning (sometimes they can also mean match beginning/end of line which may give us slightly different semantics.